python/ray/dashboard/client/src/pages/dashboard/logical-view/Actor.tsx

```diff
@@ -168,11 +181,8 @@ class Actor extends React.Component<Props & WithStyles<typeof styles>, State> {
         </Typography>
         {actor.state !== -1 && (
           <React.Fragment>
-            {actor.webuiDisplay && (
-              <Typography className={classes.webuiDisplay}>
-                {actor.webuiDisplay}
-              </Typography>
-            )}
+              {actorCustomDisplay && <Typography className={classes.webuiDisplay}>Actor Custom Display</Typography>}
```

Comment:
If `actorCustomDisplay` is a list, then it always evaluates to `true` in a boolean expression (see https://stackoverflow.com/a/19146220). We should instead check that `actorCustomDisplay.length > 0`.